src/satellitePropagator/physics/AccelerationComponents/DragComponent.h

```diff
@@ -5,15 +5,130 @@
 #pragma once
 #include "satellitePropagator/debris/Debris.h"
 #include "satellitePropagator/physics/Constants.h"
+#include <map>
 
 namespace Acceleration::DragComponent {
-namespace {
-}
+/**
+ * Table with values for density in different altitudes.
```

Comment:
We might want to mention that this is from https://en.wikipedia.org/wiki/NRLMSISE-00 (we should also cite that in the paper I guess)

Reply:
done a62977c785766aa302da66f86836df0682f003c4